Glycine/max/wip_marlene/Pan_Yu_2021

```diff
@@ -0,0 +1,18 @@
+gene_symbols:
+  - EcABCC8
+gene_symbol_long: Echinocloa colona ATP-binding cassette transporter
+gene_model_pub_name: MT249005
+gene_model_full_id: MT249005
+confidence: 5
+comments: 
+  - Overexpression of EcABCC8 (or its orthologs) in transgenic plants created glyphosate resistance. EcABCC8 knockout plants had decreased resistance to glyphosate.
+phenotype_synopsis: EcABCC8 codes for an efflux pump that 'extrudes' the herbicide glyphosate out of a cell's cytoplasm, making Echinochloa colona ('awnless barnyardgrass') glyphosate resistant.
+traits:
+  - entity_name: herbicide resistance
```

Comment:
I couldn't find an ontology tag specific to glyphosate resistance.  Is this a viable alternative label?

Reply:
herbicide resistance would be OK if you can find that in a suitable ontology - but it looks like this one in TO will work: [glyphosate sensitivity](https://www.ebi.ac.uk/ols4/search/TO_0005006)

Reply:
Should I replace the traits I have listed with glyphosate sensitivity or add glyphosate sensitivity to the list?

Reply:
I think: add "glyphosate sensitivity" as the first in the set of ontology terms (TO then OMIT then GO). Order is not conserved by all yaml-processing software, but it is by some applications; and I think it's best to use the general terms from TO and PO first.